.github/workflows/test-build-release.yml

```
@@ -74,7 +74,8 @@ jobs:
 
   create-release:
     name: Create release
-    needs: test-build
+    needs: 
+      - test-build
```

It only needs one thing, so the list can be omitted:
```suggestion
    needs: test-build
```